templates/superset_config.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+
+secret = os.getenv("SUPERSET_SECRET_KEY")
+if secret:
+    SECRET_KEY = secret
```

Comment:
Can we have a problem with accessing `SECRET_KEY` when it was not defined? I am cautious about conditionally defined variables.

Reply:
The `superset_config.py` is used to overwrite the default values as defined by superset. If `SECRET_KEY` is not defined here then superset's default will be used. 

 That being said, it should not be possible for `SUPERSET_SECRET_KEY` to be a falsey value as the value is set with `self.config.get("superset-secret-key") or generate_password()`. So I've removed the conditionality, and added a note to the config value description.